src/main/java/org/scream3r/jssc/SerialPortList.java

```diff
@@ -59,6 +59,11 @@
                 PORTNAMES_PATH = "/dev/";
                 break;
             }
+            case SerialNativeInterface.OS_FREEBSD: {
+                PORTNAMES_REGEXP = Pattern.compile("(cuaU)[0-9]{1,3}");
```

Comment:
This pattern should be "(cuaU)[0-9]{1,3}$" to pull in only the base cuaU* nodes and not the .init/.lock devices.